README.md

```diff
@@ -1,218 +1,84 @@
-<img src="https://storage.googleapis.com/passage-docs/passage-logo-gradient.svg" alt="Passage logo" style="width:250px;"/>
+![passage-ruby](https://storage.googleapis.com/passage-docs/github-md-assets/passage-ruby.png)
 
-[![Gem Version](https://badge.fury.io/rb/passageidentity.svg)](https://badge.fury.io/rb/passageidentity)
+![Gem Version](https://img.shields.io/gem/v/passageidentity)
```

Comment:
I think we're putting these shields on a single line (or at least I have been!)

Reply:
I have been as well - if you're looking at the preview you're probably seeing the old badge.fury shield I removed above the news

Reply:
Top one is showing what was removed, the bottom three are the actual ones that will be there: 
<img width="399" alt="image" src="https://github.com/user-attachments/assets/4161d221-8b1b-4ba0-aee7-daaa82e30cbc">

Reply:
Whoops you're right, didn't realize I was in unified view!